src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue

```diff
@@ -112,6 +112,11 @@ export default {
 			type: String,
 			default: '',
 		},
+
+		subtitleText: {
```

Comment:
ideally we could get the text from the slot contents but this turned out to be a non-trivial problem especially with Vue 3